src/main/java/com/microsoft/azure/batch/TaskOperations.java

```diff
@@ -93,18 +94,19 @@ public void createTasks(String jobId, List<TaskAddParameter> taskList) throws Ru
     }
 
     private static class WorkingThread implements Runnable {
-
-        final int MAX_TASKS_PER_REQUEST = 100;
-
-        private BatchClient client;
-        private BehaviorManager bhMgr;
-        private String jobId;
-        private Queue<TaskAddParameter> pendingList;
-        private List<TaskAddResult> failures;
-        private volatile Exception exception;
-        private final Object lock;
-
-        WorkingThread(BatchClient client, BehaviorManager bhMgr, String jobId, Queue<TaskAddParameter> pendingList, List<TaskAddResult> failures, Object lock) {
+    	final static int MAX_TASKS_PER_REQUEST = 100;
+    	private static final AtomicInteger currentMaxTasks = new AtomicInteger(MAX_TASKS_PER_REQUEST);
+
+    	private BatchClient client;
+    	private BehaviorManager bhMgr;
+    	private String jobId;
+    	private Queue<TaskAddParameter> pendingList;
+    	private List<TaskAddResult> failures;
+    	private volatile Exception exception;
+    	private final Object lock;
+
+		WorkingThread(BatchClient client, BehaviorManager bhMgr, String jobId, Queue<TaskAddParameter> pendingList,
```

Comment:
Fix intent

Reply:
Forgot to update my java IDE editor settings